rasa/shared/nlu/training_data/formats/readerwriter.py

```diff
@@ -33,17 +33,18 @@ def _raise_on_same_start_and_different_end_positions(
     Args:
         aggregated_entities: Entities for each start position
     """
-    for entity_list in aggregated_entities.values():
-        end = entity_list[0]["end"]
-        for entity in entity_list[1:]:
-            # By construction, start positions of all entities in `entity_list` are
-            # identical
-            if entity["end"] != end:
-                raise ValueError(
-                    f"Entities '{entity}' and "
-                    f"'{aggregated_entities[entity['start']][0]}' have identical "
-                    f"start but different end positions"
-                )
+     for entity_list in aggregated_entities.values():
+        if len(entity_list) > 0:
```

Comment:
Did you run into an error where the list was empty? As far as I can see it wasn't previously necessary to add this check